app/models/account.rb

```diff
@@ -74,6 +74,9 @@ def self.columns
 
   scope :not_master, -> { where(master: [0, nil]) }
 
+  scope :paid, -> { where(paid: true) }
+  scope :free, -> { where(paid: false) }
```

Comment:
I am not sure about this yet because we have an attribute `paid` but then for the instance the current way of checking if it is paid or not is this one instead:
https://github.com/3scale/porta/blob/9eba252eada057e680d97dab98b844c3a5c80c43/app/models/account.rb#L528-L530

Reply:
I think the real way is the `def paid?`

Reply:
For everyone: I asked Hery in Slack:

> I am not sure if for security checking if one of both is true and then considering it as a PAID account with an or, like:
> it is paid if it either has any paid contract or if it has the paid attribute as true.
> WDYT?
 
And he said that he doesn't know where that attribute is updated and we should find it 😄

Reply:
Ok... clearly not used anymore for anything 😝 
![image](https://user-images.githubusercontent.com/11318903/51755953-1fc00b80-20c0-11e9-857c-c31ca1368759.png)

Reply:
Another column that is not used at all ...